qiskit/ignis/mitigation/dynamical_decoupling/xy4.py

```diff
@@ -0,0 +1,158 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""XY4 is a basic dynamical decoupling (DD) sequence that applies corrective pulses around
+both the X and Y axes, to correct multiple components of the system-environment interaction.
+The sequence is comprised of an X gate, Y gate, X gate, and Y gate with a fixed delay between
+each pulse.
+
+This implementation saturates idle periods of any qubit with XY4 sequences.
+"""
+from qiskit import circuit
+from qiskit.circuit.delay import Delay
+from qiskit.converters import circuit_to_dag
+
+from qiskit.transpiler.basepasses import TransformationPass
+from qiskit.transpiler.passes.basis.unroller import Unroller
+
+
+class XY4(TransformationPass):
+    """Pass which inserts XY4 sequences into a scheduled circuit where there are delays
+    of sufficent duration.
+    """
+
+    def __init__(self, backend_properties, tau_step=10e-9):
```

Comment:
Is there any reason for the default `tau_step` of 10 ns? I think this value is too specific, i.e. 10 ns is also valid for ion trap? Perhaps some ratio to the pi pulse?